_posts/2018-04-25-beta-release.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.0 Core Beta Release
```

Comment:
What about adding a '1' on the end to suggest it's numbered and the first.

Reply:
"Beta Release 1" or "Beta 1 Release"?

Reply:
Beta 1 Release